### PR TITLE
Added docs for telegram cloud plugin.

### PR DESCRIPTION
**Pull request by AlexanderGerasimovich**

### CODE REVIEW

File: telegram/cloud/overview.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<h2>Product Overview</h2>
+
+<p><b>In this quick overview we will go over basic functionality of plugin.</b></p>
+<p>
+<ol type="1">
+    <li><b>What features does the plugin provide?</b>
+    <ul>
+        <li>Create telegram chats directly from JIRA issue view or project screen.</li>
+        <li>Add to telegram chat new members, remove existing chat participants, manage admin privileges.</li>
+        <li>Receive notifications about issue status from telegram bot.</li>
```
Comment (itransition-atlassian) [inline telegram/cloud/overview.html]: `Telegram bot`

File: telegram/cloud/overview.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <li>Post messages from telegram chat to linked issue in Jira.</li>
```
Comment (itransition-atlassian) [inline telegram/cloud/overview.html]: please write `Telegram` first letter with upper case

File: telegram/cloud/user-registration.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                 title="Click the image to open in full size"
+                 src="../../assets/img/screenshots/telegram/cloud/bot-start-command.png"/>
+        </a>
+    </li>
+</ol>
+<br>
+<p>That's it! The Plugin is configured now!</p>
+<p><b>Note:</b> An Admin account and Bot are required for the plugin to work. If you press <b>"Log out"</b> button, <b>the plugin will stop working!</b> And most importantly, <b>do not remove the bot!</b></p>
+
+
```
Comment (itransition-atlassian) [inline telegram/cloud/user-registration.html]: please remove extra lines

File: telegram/cloud/admin-guide.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    <li>
+        Follow BotFather instructions.
+        <br>
+    </li>
+    <li>
+        Receive a message from BotFather with your new bot data.
+        <br>
+    </li>
+</ol>
+<p><b>User authorization:</b></p>
```
Comment (itransition-atlassian) [inline telegram/cloud/admin-guide.html]: `Admin User Authorization:`

File: telegram/cloud/admin-guide.html [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+<ol type="1">
+    <li>
+        After installing the plugin click on "Configure" or "Global Settings".
```
Comment (itransition-atlassian) [inline telegram/cloud/admin-guide.html:42]: do we have `Configure` page??

File: telegram/cloud/admin-guide.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        <br>
+    </li>
+    <li>
+        <p>Then you need to log in to your Telegram account.
+        You will be authenticated as an admin, so your account will be used for sending confirmation messages during other users registration.
```
Comment (itransition-atlassian) [inline telegram/cloud/admin-guide.html]: >> your account will be used for sending confirmation messages during other users registration

what does this mean? do we really use admin account to send any messages?

File: telegram/cloud/admin-guide.html [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+        <a href="#" onclick="return false;">
+            <img onclick="makeFullScreen(this)" class="screenshot-small fullscreen"
+                 title="Click the image to open in full size"
+                 src="../../assets/img/screenshots/telegram/cloud/bot-start-command.png"/>
+        </a>
+    </li>
+</ol>
+<br>
+<p>That's it! The Plugin is configured now!</p>
+<p><b>Note:</b> An Admin account and Bot are required for the plugin to work. If you press <b>"Log out"</b> button, <b>the plugin will stop working!</b> And most importantly, <b>do not remove the bot!</b></p>
```
Comment (itransition-atlassian) [inline telegram/cloud/admin-guide.html]: if admin log out, will all other users stop receiving the notifications?

File: telegram/cloud/bot-abilities.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
+---
+layout: telegram-cloud-doc
+title: Telegram Integration for Jira Cloud
+---
+<h2>User guide. Part II. Bot abilities</h2>
+<p><b>Why do we need a bot?</b></p>
+<p>Gives you a wonderful opportunity to manage your projects via the popular messenger <b>Telegram</b> which makes your work much more convenient.</p>
```
Comment (itransition-atlassian) [inline telegram/cloud/bot-abilities.html]: `Bot gives you...`

File: telegram/cloud/bot-abilities.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+<p>You can create <b>project chats</b> to monitor all activity in general on your project,
+    which can help you to spot all bottlenecks of your project, so you could manage them.</p>
+<p>You can also create <b>issue chats</b> to control some specific, maybe especially important tasks.</p>
+<p><b>Jira Bot</b> will notify you about everything you decide is important! If an issue or chat for an issue was deleted, the bot sends a notification to this chat and leaves it.</p>
```
Comment (itransition-atlassian) [inline telegram/cloud/bot-abilities.html]: please check my comment in the wiki

**Review by itransition-atlassian — CHANGES_REQUESTED** [review]

File: telegram/cloud/admin-guide.html (hunk 5 above)
Comment (AlexanderGerasimovich) [reply telegram/cloud/admin-guide.html:42]: The 'Configure' button and the 'Global Settings' link lead to the same configuration page.

File: telegram/cloud/admin-guide.html (hunk 6 above)
Comment (AlexanderGerasimovich) [reply telegram/cloud/admin-guide.html:47]: Only an administrator can access the Configure page.

**Review by itransition-atlassian — APPROVED** [review]

**Pull request merged**